storage_service/storage_service/settings/base.py

```diff
@@ -86,7 +86,6 @@ def _get_settings_from_file(path):
 USE_I18N = True
 
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#use-l10n
```

Comment:
Should we delete this line?

Reply:
yes we can as per documentation this feature is added by default in Django 4.2